src/main/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/PileupDetectionArgumentCollection.java

```diff
@@ -64,14 +106,17 @@ public final class PileupDetectionArgumentCollection {
     public double snpThreshold = 0.1;
     @Hidden
     @Argument(fullName= PILEUP_DETECTION_INDEL_THRESHOLD, doc = "Pileup Detection: Percentage of alt supporting reads in order to consider alt indel. (Requires '--"+PILEUP_DETECTION_LONG_NAME+"' argument)", optional = true)
-    public double indelThreshold = 0.5;
+    public double indelThreshold = 0.1;
```

Comment:
because i care not about how the old code pileup code works for indels because it was not part of the default path and was not finished when i picked up this branch.